app/views/static/about.html.erb

```diff
@@ -1,4 +1,4 @@
-
+<% @page_title = "#{t('about')} - #{application_name}" %>
```

Comment:
I'm not sure I understand how `@page_title` is being used. For the About and Contact pages, it looks like the breadcrumbs are using `title`, defined on the next line.

Reply:
It sets the the html title tag in the head, so that the tab in Chrome shows "Digital Repository Contacts - Emory Digital Collections" on the contact page.

Reply:
In `blacklight_helper_behavior`:
```# Get the page's HTML title
  #
  # @return [String]
  def render_page_title
    (content_for(:page_title) if content_for?(:page_title)) || @page_title || application_name
  end```

Reply:
Also, I tried to add as many tests for the material I added, but "total-coverage" still went down. Should I keep on writing tests?